examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -56,6 +56,7 @@
 
 from ansys.mapdl import core as pymapdl
 
+pv.OFF_SCREEN = True
```

Comment:
I think this is not needed, because it is on the ``conf.py``, but I guess it was part of your tests. Can we remove it?

Reply:
Done.